### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by Rian501**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+article {
+	margin: 12px;
+}
+
+/************************************
+NAV STYLES
+************************************/
+nav ul {
+display: flex;
```
Comment (gregkorte) [inline css/styles.css:27]: Watch your indentation.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		<label>Connect with me:</label><br>
+		<section class="soc-med-icons">
+			<a href="mailto:eclaire.lemmon@gmail.com">
+			<img class="soc-med-icon" class="linkedin" src="../images/mail_circle.png" alt="email logo">
+			</a>
+
+			<a href="https://www.linkedin.com/in/emily-c-lemmon-59479513/">
+			<img class="soc-med-icon" class="linkedin" src="../images/linkedin.png" alt="linkedin logo">
+			</a>
+
```
Comment (gregkorte) [inline index.html:77]: Whitespace.

**Comment by gregkorte** [comment]
Nice work, Emily! Other then an indentation issue and some whitespace, this challenge looks like it's good to merge! :+1:

**Comment by Rian501** [comment]
Is there a way to change the files right on github if there is no conflict? I know I can access it seemingly automatically if there are merge conflicts, but I'm not sure how to do it if there aren't.

**Pull request merged**